web/profiles/pece/module/pece_access_permission/config/install/core.entity_form_display.paragraph.paragraph_members.default.yml

```diff
@@ -0,0 +1,31 @@
+uuid: b0b05102-576a-4634-bd76-2bbd553ed893
```

Comment:
Para colocar na pasta install vc precisa ter configurado e exportando essa propriedade no config devel para dentro da pasta config/dev, se não é melhor apagar da pasta install

Reply:
@edmargomes eu exportei pra pasta config/sync. Não entendi pq tinha que ser na config/dev

Reply:
Esse arquivo não está na pasta config/sync, seta na config install.

Se for usar a config/install é necessário configurar a pasta desse install no modulo config_devel e exportar essa nova config para a pasta config/dev